packages/openapi3/test/tsp-openapi3/output/petstore-swagger/main.tsp

```diff
@@ -15,10 +15,8 @@ using OpenAPI;
  * - [The Pet Store repository](https://github.com/swagger-api/swagger-petstore)
  * - [The source API definition for the Pet Store](https://github.com/swagger-api/swagger-petstore/blob/master/src/main/resources/openapi.yaml)
  */
-@service({
-  title: "Swagger Petstore - OpenAPI 3.0",
-})
-@info({
+@service(#{ title: "Swagger Petstore - OpenAPI 3.0" })
+@info(#{
   version: "1.0.20-SNAPSHOT",
   contact: {
```

Comment:
I think the nested objects were missed - though I'm not sure why given the conversion code looks correct.

Reply:
the ci seems stuck so might have missed things